src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/ServiceHearingsControllerTest.java

```diff
@@ -0,0 +1,107 @@
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseDetails;
+import uk.gov.hmcts.reform.sscs.ccd.service.CcdService;
+import uk.gov.hmcts.reform.sscs.exception.AuthorisationException;
+import uk.gov.hmcts.reform.sscs.idam.IdamService;
+import uk.gov.hmcts.reform.sscs.idam.IdamTokens;
+import uk.gov.hmcts.reform.sscs.service.AuthorisationService;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.willDoNothing;
+import static org.mockito.BDDMockito.willThrow;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.reform.sscs.service.AuthorisationService.SERVICE_AUTHORISATION_HEADER;
+
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+@ActiveProfiles("integration")
+public class ServiceHearingsControllerTest {
+
+    private static final String CASE_ID = "1625080769409918";
+    private static final String AUTHORIZATION = "1203912-39012-=391231";
+    private static final String CASE_CREATED_DATE = "05/11/2021";
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private IdamService idamApiService;
+
+    @MockBean
+    private AuthorisationService authorisationService;
+
+    @MockBean
+    private AuthTokenGenerator authTokenGenerator;
+
+    @MockBean
+    private CcdService ccdService;
+
+    @BeforeEach
+    public void setUp() {
+
+        SscsCaseDetails caseDetails =
+                SscsCaseDetails.builder().data(SscsCaseData.builder().caseCreated(CASE_CREATED_DATE).build()).build();
+        given(ccdService.getByCaseId(anyLong(), any(IdamTokens.class))).willReturn(caseDetails);
+        given(authTokenGenerator.generate()).willReturn("s2s token");
+        given(idamApiService.getIdamTokens()).willReturn(IdamTokens.builder().build());
+        willThrow(new AuthorisationException(new Exception("Test"))).given(authorisationService).authorise(anyString());
+        willDoNothing().given(authorisationService).authorise(AUTHORIZATION);
+    }
+
+    @DisplayName("Should return the case creation date with a with 200 response code")

```

Comment:
Do these override the name displayed on the test report? If so it might be worth adding in why it should return x response based on the request.